docs/build/js-client/10-marine-js.md

```diff
@@ -2,11 +2,36 @@
 
 > This section is experimental as the API is not stable and probably will change.
 
+## What is Marine JS
+
+### Intro
+When JS client starts up, it starts with initialization Marine JS runtime.
+The runtime hosts wasm services and even allows you to register your own wasm services.
+For example, AquaVM service for processing particles and Air code resides in Marine JS.
+That's why Marine JS is a foundation of JS client. 
+
+### Use cases
+
+> To register your own service, you would usually use JS-client js services as they are simpler to write and implement.
+
+Usually you don't need to interact with Marine JS directly. 
+AquaVM service being registered at JS client startup phase and ready to process your compiled Aqua code.
+
+So if your needs include any of the following:
+- Cross-platform service which works both on Nox peer and JS client peer
```

Comment:
i don't get the cross-platform issue working on both the rust and js peers.

Reply:
By default js client services non cross-platform. I.e. you cannot deploy them on nox

Reply:
Most of the times `cross-platform` means linux-macos-windows. I don't see a point to use this wording here at all, since the rest of the sentence explains it anyway. So seems like it can be safely removed